src/components/ProgramProfileComponents/Charts/ClassroomClimateBarDetails.tsx

```diff
@@ -376,8 +376,8 @@ class ClassroomClimateBarDetails extends React.Component<Props, {}> {
         </div>
         <div
           style={{
-            //width: '100%',
-            width: 300 + this.state.teacherNames.length *160,
+            width: width,
+            //width: 300 + this.state.teacherNames.length *160,
             display: 'flex',

```

Comment:
Want to just delete the comment at this point? Or are we unsure of whether we want to keep it?